plugin/phpactor.vim

```diff
@@ -0,0 +1,4 @@
+augroup phpactor
+  autocmd!
+  autocmd FileType php call phpactor#project#bufferMatcher#assignFileToProject(expand('<afile>'))
+augroup END
```

Comment:
Should this be moved to the `ftplugin` folder?

Reply:
It seems to work.